appconfiguration/azure-spring-cloud-feature-management/dynamic-features-sample/src/main/java/com/azure/spring/dynamicfeatures/ApplicationConfiguration.java

```diff
@@ -0,0 +1,19 @@
+package com.azure.spring.dynamicfeatures;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Scope;
+
+import com.azure.spring.cloud.feature.manager.feature.filters.TargetingEvaluator;
+import com.azure.spring.cloud.feature.manager.targeting.TargetingEvaluationOptions;
+import com.azure.spring.dynamicfeatures.implementation.TargetingContextImpl;
+
+@Configuration
+public class ApplicationConfiguration {
+
+    @Bean(name = "Microsoft.Targeting")
```

Comment:
In normal case, bean name will like `microsoftTargeting`

Reply:
This is specific because the name is set from the App Configuration service, and isn't modifiable, so it is the same name as it would be in DotNet.